pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/OncHistoryDetail.java

```diff
@@ -350,7 +313,7 @@ public static OncHistoryDetail getOncHistoryDetail(@NonNull ResultSet rs) throws
     }
 
     // TODO: there should be no need for the realm parameter -DC
-    public static OncHistoryDetail getOncHistoryDetail(@NonNull int oncHistoryDetailId, String realm) {
```

Comment:
`@NonNull` makes no sense on atomics, and in general add no/little value. If a parameter value can be null, that should be part of the method JavaDocs.